subsys/net/ip/Kconfig.ipv6

```diff
@@ -39,6 +39,14 @@ config NET_IF_IPV6_PREFIX_COUNT
 
 if NET_NATIVE_IPV6
 
+config NET_IPV6_MTU
+	int "Initial IPv6 MTU value"
+	default 1280
+	range 1280 1500
```

Comment:
nit: what about to allow higher MTUs (there are physical layers that could go above 1500).. but i don't have a particular use-case for now.

Reply:
We can tweak the values if/when needed and use case arises.